sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2408,39 +2396,43 @@ public virtual async Task<AnalyzeActionsOperation> StartAnalyzeActionsAsync(IEnu
 
         private AnalyzeActionsOperation StartAnalyzeActions(MultiLanguageBatchInput batchInput, TextAnalyticsActions actions, AnalyzeActionsOptions options = default, CancellationToken cancellationToken = default)
```

Comment:
The diff looks all weird. what I did was:
- Create method `ValidateActions` that throws if more than one action type is set
- Aggregated all transforms from actions to tasks into a method called `CreateTasks` that can be used by both `StartAnalyzeActions` and `StartAnalyzeActionsAsync` methods